tests/AulaTests/Stories/DSL.hs

```diff
@@ -58,6 +59,9 @@ selectIdeaSpace n = liftF $ SelectIdeaSpace n ()
 createIdea :: IdeaTitle -> IdeaDescription -> Category -> Behavior ()
 createIdea title desc cat = liftF $ CreateIdea title desc cat ()
 
+editIdea :: IdeaTitle -> IdeaTitle -> IdeaDescription -> Category -> Behavior ()
+editIdea oldTitle newTitle desc cat = liftF $ EditIdea oldTitle newTitle desc cat ()
```

Comment:
It depends on the interpretation. The actual implementation picks the first one for the `Action` interpreter. The test suite has to designed the way, to avoid that collusion and test against them on a different level.